doc/distrib/GoogleAnalyticsConsent.rtf

```diff
@@ -1,197 +0,0 @@
-{\rtf1\adeflang1025\ansi\ansicpg1252\uc1\adeff37\deff0\stshfdbch0\stshfloch37\stshfhich37\stshfbi37\deflang18441\deflangfe2052\themelang1033\themelangfe0\themelangcs0{\fonttbl{\f0\fbidi \froman\fcharset0\fprq2{\*\panose 02020603050405020304}Times New Roman;}
```

Comment:
Can't remove this file for now, this is preserved for integrator patch installer